### DISCUSSION TITLE
New release when? It's been almost 2 years!

### DISCUSSION THREAD

**Discussion by gnarlin** [flareteam/flare-game#972] [General]
Hello.
It's been almost 2 years now since the last release. I checked the commit history and while the game itself only has sporadic commits the game engine's development seems fairly healthy, or am I mistaken? Isn't it about time to make a new release with some fanfare? Surely there have been some new features, bugs fixes, even a few game improvements and updates?
Could we please at least get a best guestimation on a future release?
Has there, perhaps, already been a giant announcement or goal check list somewhere and I just completely missed it?

Anyway, good game. 
Thank you.

**Comment by dorkster** [comment]
I'll admit there's no rush for version 1.15, and I've been treating it as "when it's done". There's a couple people (myself included) that are working on stand-alone Flare mods. Working on my own mod and helping others has shed light on many bugs that were either in 1.14, or introduced in the current development cycle.

But the [fixes and features](https://github.com/flareteam/flare-engine/blob/master/RELEASE_NOTES.txt) have been piling up and the items on the [roadmap](https://github.com/orgs/flareteam/projects/1/views/1) are dwindling. So I think a release is nearing readiness.

I think the soonest I'd say for a release would be **early 2025**.

**Comment by gnarlin** [comment]
Great! Thank you for those links. I look forward to 2025 :D
